beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/utils/BeekeeperApiTestClient.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) 2019-2021 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.integration.utils;
+
+import static java.net.http.HttpRequest.newBuilder;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+
+public class BeekeeperApiTestClient {
+
+  private static final String API_ROOT = "/api/v1";
+  private static final String DATABASE_AND_TABLE = "/database/some_database/table/some_table";
```

Comment:
Why is this path hardcoded? The db name and table name should be variables that should be put in to this string

Reply:
Since this is used for testing, I always use the same database name and table name, as the only time I would need a different one is to check what happens when a table is not found, and this is done here: https://github.com/ExpediaGroup/beekeeper/blob/a225d25ecd2f7f56f9af98ce4eb286d313ce2d51/beekeeper-integration-tests/src/test/java/com/expediagroup/beekeeper/integration/api/BeekeeperApiIntegrationTest.java#L99

Reply:
But for future proofing and just because this is a client, can you make it generic please?